### PR TITLE
Docs: Nsight-Systems

### PR DESCRIPTION
**Pull request by ax3l**
How to use Nsight-Systems.
Tested on Cori DGX.

### CODE REVIEW

File: Docs/source/developers/profiling.rst [hunk 1]
```diff
@@ -3,5 +3,54 @@
...
 
-.. note::
-   Section empty!
+.. _developers-profiling-tiny-profiler:
+
+Tiny Profiler
+-------------
+
+The so-called "Tiny Profiler" is a component of lightweight profiling that we perform with each WarpX run by default.
+A table of the summary of most often called and most costly functions is printed to ``stdout`` at the end of every WarpX run.
```
Comment (roelof-groenewald) [inline Docs/source/developers/profiling.rst:12]: @MKieburtz Could you please suggest a sentence to add here describing how a user could employ your post-processing script to parse the Tiny Profiler output. Thanks!

Comment (EtherealMC-Bit) [reply Docs/source/developers/profiling.rst:12]: Yep!

**Comment by EtherealMC-Bit** [comment]
Suggested sentence to add here about the TinyProfile parser.
> There is a script located [here](https://github.com/AMReX-Codes/amrex/tree/development/Tools/TinyProfileParser) that parses the Tiny Profiler output and generates a JSON file that can be used with [Hatchet](https://hatchet.readthedocs.io/en/latest/) in order to analyze performance.

Does this sound good?

**Comment by ax3l** [comment]
Love it - and it has nice API and argument docs. That makes me very happy.

If you like to describe a typical workflow how you use Hatchet in WarpX development, don't hesitate add more info in this chapter of the manual.

**Review by roelof-groenewald — APPROVED** [review]

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**